README.md

```diff
@@ -1103,14 +1103,20 @@ column type.
 #### Convert a column from text to json
 
 Postgres' `alter column` command will not automatically convert a `text`
-column to `json`, but it can still be done with plain SQL.
+column to `json`, but it can still be done with a piece of plain SQL.
 
-```sql
-alter table versions
-alter column object type json
-using object::json;
+```ruby
+def up
+  change_column :versions, :object, 'jsonb USING object::jsonb'
+end
+
+def down
+  change_column :versions, :object, 'text USING object::text'
+end
 ```
 
+You'll have to destroy or migrate all `versions` data before running this migration.
```

Comment:
You inserted JSON into text column and converted it. But with paper_trail we actually have YAML in this column, and PostgreSQL can't guess what it is and convert it.